### PR TITLE
Fix date format on certificates generation

### LINKED ISSUE #8859: Fix date format on certificates
We need to change the date format on our certificates. Currently it's 
January, 15th 2016
It should be:
January 15, 2016

Note that the comma needs to be moved, and the date should just be a natural number, not an ordinal number.

### PR DESCRIPTION
**Pull request by scottmcallister**
#### Pre-Submission Checklist
- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change
- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:
- [x] Tested changes locally.
- [x] Closes #8859
#### Description

Changed the date format for certificate from ordinal number to natural number. The comma was already in the right place.

### CODE REVIEW

File: server/boot/certificate.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ function getIdsForCert$(id, Challenge) {
 // getFormatedDate(challengeMap: Object, challengeId: String) => String, throws
 function getFormatedDate(challengeMap, challengeId) {
   return moment(challengeMap[challengeId].completedDate)
-    .format('MMM Do, YYYY');
+    .format('MMM D, YYYY');
```
Comment (raisedadead) [inline server/boot/certificate.js]: Update this to the correct format as requested in the issue.
Refer : http://momentjs.com/docs/#/displaying/format/

Comment (raisedadead) [inline server/boot/certificate.js]: Shouldn't this be `MMMM` ?

**Comment by scottmcallister** [comment]
You're right @raisedadead, the format for the month should be MMMM.  I've updated this PR to include this change, although I see someone else has already submitted another PR for this issue.

**Comment by raisedadead** [comment]
@scottmcallister thanks for making the change, and yes there is another PR #8862 but that doesn't close the issue completely, thing is we need both the PR's to address the issue (sort of) :sweat_smile: !

So I have updated some tags in the other PR to reflect the same.

**Pull request merged**